tests/dotnet-format.UnitTests.csproj

```diff
@@ -1,10 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
did we mean to roll this back?

Reply:
Yeah, since we are rolling forward I wanted running the unit tests to behave the same a running dotnet-format.

Reply:
the roll forward _should_ allow this to work without issue